plaso/cli/psort_tool.py

```diff
@@ -239,9 +240,16 @@ def _ParseProcessingOptions(self, options):
 
     if worker_memory_limit and worker_memory_limit < 0:
       raise errors.BadConfigOption(
-          'Invalid worker memory limit value cannot be negative.')
+          'Invalid worker memory limit value cannot be less than 0.')
+
+    worker_timeout = getattr(options, 'worker_timeout', None)
+
+    if worker_timeout is not None and worker_timeout <= 0.0:
+      raise errors.BadConfigOption(
+          'Invalid worker timeout value must be larger than 0.0 minutes.')
```

Comment:
Please include the invalid value in the error message here: "Invalid memory limit -1. Limit cannot be less than 0."

Reply:
done